bin/test

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+cookiecutter \
+  --no-input \
+  --config-file bin/test-config.yml \
+  .
+
+cd ./TestProject
+
+bin/test
```

Comment:
Do you have a name suggestion? I'm not sure that there's anything I can think of that would be less confusing.